test/Math/Rectangle.test.cpp

```diff
@@ -61,6 +61,34 @@ TEST(Rectangle, translate) {
 	EXPECT_EQ((NAS2D::Rectangle{4, 6, 5, 6}), (NAS2D::Rectangle{3, 4, 5, 6}).translate({1, 2}));
 }
 
+TEST(Rectangle, operatorAddSubtract) {
+	//translate(...) and operator overloads are equivalent.
```

Comment:
Comment typically have a space after the `//`:
```cpp
// Translate and operator overloads are equivalent
```

Also, I tend to avoid periods at the end of comments, unless maybe it's a block paragraph.